src/css.js

```diff
@@ -273,7 +274,7 @@ jQuery.extend({
 
 			// Convert "+=" or "-=" to relative numbers (#7345)
 			if ( type === "string" && (ret = rrelNum.exec( value )) ) {
```

Comment:
Optimization opportunity (i.e., apply if it yields a smaller jquery.min.js.gz): get rid of `rrelNum` completely by using `rfxnum` in its place here: `if ( type === "string" && (ret = rfxnum.exec( value )) && ret[ 1 ] )`.